vspace/src/main/webapp/WEB-INF/views/staff/user/list.jsp

```diff
@@ -0,0 +1,60 @@
+<%@ page pageEncoding="UTF-8" %>
```

Comment:
why is this file here?

Reply:
Actually, these files are not part of my story. I merge the latest code changes of ```VSPC-120``` and then re-open PR, which is why I don't change any formatting if any. Is there any way by which I can see what line endings or files to be removed after merging another ticket's code changes?

Reply:
You're adding this file, so it cannot have been in VSPC-120. same with space.jsp.

Reply:
Okay. So should I delete `list.jsp` and `space.jsp`. Since I am not using them?

Reply:
you shouldn't add them (which means you need to delete them from your branch).

Also, please reread the pull request guidelines. This is something you should have noticed before I reviewed it.

Reply:
Thanks Julia, I will re-read them. But, I did a merge with `VSPC-120` and added files that come on staging, not sure how did they come up. I will be careful next time.